core/templates/pages/skill-editor-page/services/skill-editor-routing.service.ts

```diff
@@ -64,6 +64,17 @@ angular.module('oppia').factory('SkillEditorRoutingService', [
       },
       navigateToPreviewTab: function() {
         $location.path('/preview');
+      },
+      // To navigate directly to question-editor interface
+      // from skill editor page.
+      createNewQuestion: function() {
+        ctrl.createQuestion = true;
+      },
+      createQuestion: function() {
+        return ctrl.createQuestion;
+      },
+      backToQuestionList: function() {
+        ctrl.createQuestion = false;
```

Comment:
From what I can tell `createNewQuestion` and  `backToQuestionList` sets/unsets a flag and `createQuestion` just returns the value of the flag. If so can we rename them accordingly? Also, maybe just one "setter" method is enough.

Reply:
Done.